src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -469,6 +531,34 @@ public void ResetResolution()
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Requests to capture this SceneView with the Camera.
+        /// </summary>
+        /// <param name="camera">Camera to be used for capture.</param>
+        /// <param name="size">captured size.</param>
+        /// <remarks>
+        /// The input camera should not be used for any other purpose during Capture.
+        /// The selected camera cannot be used for input camera.
+        /// The camera is required to be added in this SceneView.
+        /// If the SceneView is disconnected from Scene, the left capture request is canceled.
+        /// </remarks>
+        /// <returns> capture id that id unique value to distinguish each request.
+        /// If capture is requested during previous capture, invalid index(-1) is returned.</returns>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Capture(Scene3D.Camera camera, Vector2 size)
+        {
+            if(inCapture)
+            {
+                Tizen.Log.Error("NUI", "The previous capture request is not finished yet.\n");
+                return -1;  // invalid index
+            }
```

Comment:
```suggestion
            }
            if (camera == null)
            {
                Tizen.Log.Error("NUI", "Invalid Camera used.\n");
                return -1;
            }
```

Please add null check for camera

Reply:
I fixed it.